threat-model/readme.md

```diff
@@ -216,20 +216,28 @@ TODO
 
 ### 1.10 TEE Manufacturers
 
-TEEs can provide "attestation" which verifies that the TEE is running in the expected state and running the expected code. These attestations are typically produced with an asymmetric key, where the private key is physically embedded into the TEE, and the public key is published via a system similar to a certificate authority.
+The TEE manufacturer is responsible for constructing the physical TEE hardware according to a specification and managing the associated public key infrastructure (PKI).
 
 #### 1.9.1 Assets
 
-1. Private keys embedded in TEEs and their corresponding public keys.
+1. Private keys embedded in TEEs. These mush be confidential and known only to the TEE. 
+2. List of public keys corresponding to valid attestation/identity private keys embedded in TEEs. This list must be correct in that it contains only the public keys corresponding to private keys embedded in the hardware of TEEs that were manufactured according to the desired specification. If this is not the case, then valid attestations may be provided by hardware that does have the expected properties or guarantees. It must also be public so that parties other than the manufacturer are able to recognize valid TEE hardware.
```

Comment:
Is it really the case that this is a list?  There are a bunch of ways of formulating this, but I believe that the public information is generally abstracted in some way, if only because it is easier to move a small set of intermediate certificates (or equivalent) than a whole list.

Maybe a different way to approach this is to just say:

> Information about the current set of public keys that are deployed in TEEs that are considered valid.  A TEE is considered valid if it provides the properties or guarantees required of an application.  Over time, the set of TEEs that are considered valid will change as new hardware is manufactured or attacks are discovered that disqualify specific TEE implementations.  A TEE manufacturer needs to provide information or services that can enable users of TEEs to determine whether a given public key corresponds to a private key that is deployed in a TEE that is considered valid.  In practice, TEE manufacturers generally use a public key infrastructure (PKI) system for this purpose.

Reply:
Yep, list is not correct. I like your suggestion.

Reply:
made an attempt at incorporating in [09aa86c](https://github.com/patcg/docs-and-reports/pull/34/commits/09aa86c9cd622dd0625246c2ea9ca58dfd8ffcfc)